content/en/docs/reference/using-api/cel.md

```diff
@@ -61,13 +61,41 @@ Example CEL expressions:
+| CEL option, library or language feature             | Included                                                                                                        | Availablity                |
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | -------------------------- |
+| [standard macros](stdmacros)                        | `has`, `all`, `exists`, `exists_one`, `map`, `filter`                                                           | *                          |
+| [standard functions](stdlib)                        | see the [official list of standard definitions](stdlib)                                                         | *                          |
+| [Homogeneous Aggregate Literals](opt1)              |                                                                                                                 | *                          |
+| [Default UTC Time Zone](opt2)                       |                                                                                                                 | *                          |
+| [Eagerly Validate Declarations](opt3)               |                                                                                                                 | *                          |
+| [extended strings library](stringsgodoc), Version 1 | `charAt`, `indexOf`, `lastIndexOf`, `lowerAscii`, `upperAscii`, `replace`, `split`, `join`, `substring`, `trim` | *                          |
+| Kubernetes list library                             | See below                                                                                                       | *                          |
+| Kubernetes regex library                            | See below                                                                                                       | *                          |
+| Kubernetes URL library                              | See below                                                                                                       | *                          |
+| Kubernetes authorizer library                       | See below                                                                                                       | *                          |
+| [CEL Optional Values](opt4)                         | `?` operators, `optional.of`, `optional.ofNonZeroValue`, `or`, `orValue`, `optMap`, `optFlatMap`                | Kubernetes 1.29+           |
+| [CEL Cross Type Numeric Comparisons](opt5)          |                                                                                                                 | Kubernetes 1.29+           |
+| [extended strings library](stringsgodoc), Version 2 | `format`, `strings.quote`                                                                                       | Kubernetes 1.30+           |
```

Comment:
The style guide states to avoid statements about the future, since the below are for future Kubernetes versions, can they be left off until they are available?
```suggestion
| [CEL Optional Values](opt4)                         | `?` operators, `optional.of`, `optional.ofNonZeroValue`, `or`, `orValue`, `optMap`, `optFlatMap`                | Kubernetes 1.29+           |
| [CEL Cross Type Numeric Comparisons](opt5)          |                                                                                                                 | Kubernetes 1.29+           |
| [extended strings library](stringsgodoc), Version 2 | `format`, `strings.quote`                                                                                       | Kubernetes 1.30+           |
```

Reply:
Good catch. Removed.